WordPress/Classes/ViewRelated/NewGutenberg/NewGutenbergViewController.swift

```diff
@@ -469,7 +469,14 @@ class NewGutenbergViewController: UIViewController, PostEditor, PublishingEditor
 
     // MARK: - Editor Setup
     private func fetchEditorDependencies() async throws -> EditorDependencies {
-        let settings = try await blockEditorSettingsService.getSettingsString(allowingCachedResponse: true)
+        let settings: String?
+        do {
+            settings = try await blockEditorSettingsService.getSettingsString(allowingCachedResponse: true)
+        } catch {
+            DDLogError("Failed to fetch editor settings: \(error)")
+            settings = nil
```

Comment:
```suggestion
            settings = "undefined"
```

I think we wanted this literal as the fallback value? This also allows us to avoid the `settings` string having an option for `nil`.

Reply:
That is correct. However ,my understanding of the [`.apply()`](https://github.com/wordpress-mobile/WordPress-iOS/blob/19fce9c935db1ff0db2a46d9fbac54f680235c89/WordPress/Classes/ViewRelated/NewGutenberg/NewGutenbergViewController.swift#L392) was that it does _not_ set anything when `settings` is `nil`. Is that incorrect? 

If so, that would mean [GBK's `"undefined"` default](https://github.com/wordpress-mobile/GutenbergKit/blob/06a322f5fe222c4991abca407b7392f76bcff9d8/ios/Sources/GutenbergKit/Sources/EditorConfiguration.swift#L132) is used. 

Happy to change this code. Just want to make sure I understand how the Swift logic works. Let me know which you prefer.

Reply:
Oh yep, that works – probably better to have the `nil` settings in the app and leave the implementation detail of `"undefined"` in GBKit.

Thanks for explaining!!